File: app/src/main/java/com/intetm/comicbreeze/ComicApplication.java
package com.intetm.comicbreeze;

import android.app.Application;

import com.intetm.comicbreeze.service.database.DatabaseService;

public class ComicApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DatabaseService.init(getApplicationContext());
    }
}

File: app/src/main/java/com/intetm/comicbreeze/ComicApplication.kt
package com.intetm.comicbreeze

import android.app.Application
import com.intetm.comicbreeze.service.comic.ComicListService

import com.intetm.comicbreeze.service.database.DatabaseService

class ComicApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        DatabaseService.init(applicationContext)
        ComicListService.init()
    }
}
